nestkernel/recording_device.h

```diff
@@ -538,7 +538,7 @@ class RecordingDevice : public Device
     //! Store current values in dictionary
     void get( const RecordingDevice&, DictionaryDatum& ) const;
     //! Set values from dictionary
-    void set( const RecordingDevice&, const Buffers_&, const DictionaryDatum& );
+    void set( const RecordingDevice&, Buffers_&, const DictionaryDatum& );
```

Comment:
Please add the following comment here:
```c++
/* Note: `Buffers_&` cannot be `const` because `basic_ofstream::is_open()`
    is not `const` in C++98  (cf C++ Standard §27.8.1.10).
*/
```